input/witnessgen.go

```diff
@@ -47,116 +47,119 @@ type StandardWitnessType uint16
 // WitnessType interface.
 var _ WitnessType = (StandardWitnessType)(0)
 
+// NOTE: When adding a new StandardWitnessType, also update the WitnessType
+// protobuf enum in lnrpc/walletrpc/walletkit.proto, and the `allWitnessTypes`
```

Comment:
nit: maybe just use the package name here instead of full path, just personal taste ...

Reply:
yes definitely - fixed